apps/channels/models.py

```diff
@@ -159,6 +167,8 @@ def webhook_url(self) -> str:
             uri = reverse("channels:new_twilio_message")
         elif provider_type == MessagingProviderType.turnio:
             uri = reverse("channels:new_turn_message", kwargs={"experiment_id": self.experiment.public_id})
+        elif provider_type == MessagingProviderType.sureadhere:
+            uri = reverse("channels:new_sureadhere_message", kwargs={"client_id": self.extra_data.get("client_id", "")})
```

Comment:
Why use the `client_id` here instead of the `channel.external_id`?

Reply:
> However, I agree that using experiment_id

I actually suggested `channel.external_id` since that is the most direct lookup. Is there an advantage to using the experiment ID?